papaparse.js

```diff
@@ -1028,9 +1030,17 @@
 			return value;
 		}
 
-		function applyHeaderAndDynamicTyping()
+		function parseTransform(field, value) {
```

Comment:
Do we really need to create such simple function? It's not easier to use it's code directly? I do not see any other place where this code is reused, which makes me think that there is not need of a function.